sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamExecution.scala

```diff
@@ -673,7 +673,7 @@ class StreamExecution(
     if (lastExecution == null) {
       "No physical plan. Waiting for data."
     } else {
-      val explain = ExplainCommand(lastExecution.logical, extended = extended)
+      val explain = ExplainCommand(lastExecution.logical, extended = extended, streaming = true)
```

Comment:
So this means that this code will always return an updated plan for the last batch showing which data files were read instead of just referring to it as a StreamingRelation. We wouldn't have the bug if we had just used `logicalPlan` instead of `lastExecution.logicalPlan`, right? Then the problem would be that the `logicalPlan` may contain errors though?